### DISCUSSION TITLE
Errors in DPA2 finetuning process

### DISCUSSION THREAD

**Discussion by EricGaoOm** [deepmodeling/deepmd-kit#4465] [Q&A]
I used commmand  "dp --pt train input.json --finetune ../../model/OpenLAM_2.2.0_27heads_beta3.pt --model-branch H2O_H2O-PD"
, error happens when "DEEPMD INFO    Resuming from ../../model/OpenLAM_2.2.0_27heads_beta3.pt." they are " Missing key(s) in state_dict: ........" and "size mismatch for model.Domains_Alloy.atomic_model.descriptor.repformers.layers.0.linear1.matrix: copying a param with shape torch.Size([800, 128]) from checkpoint, the shape in current model is torch.Size([640, 128])." the error line is very long

**Comment by njzjz** [comment]
What version of software did you use? Where is `input.json` from?
